### PR TITLE
[Documentation][Cookbook] How to configure mailer?

### LINKED ISSUE #8065: [Cookbook] Configure Mandrill (etc.) for transactional e-mails
A lot of people asking how to use Mailjet, Mandrill to send transactional e-mail. A cookbook would be great. My solution so far was just to configure Swiftmailer and it worked like a charm. I recall also a bundle, which allowed to use Mandrill API. (Useful when hosting on Google Cloud)

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets | fixes #8065 |
| License         | MIT |

### CODE REVIEW

File: docs/cookbook/mailer.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    # The mailer host may be called "SMTP Server" for some services. Copy its name from your mailing provider and paste here.
+    mailer_host: ...
+
+    # These are Username and Password provided by the service for your account
+    mailer_user: ...
+    mailer_password: ~
+
+    # Here you provide a Mailing port suggested by your service. It can be 25, 465 or 587
+    mailer_port: 25
```
Comment (lchrusciel) [inline docs/cookbook/mailer.rst]: This parameter is not available by default

File: docs/cookbook/mailer.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+**3. **Remember to remove the** `disable_delivery: true` **parameter in the** `app/config/config_prod.yml`.
```
Comment (lchrusciel) [inline docs/cookbook/mailer.rst]: It is not 100% true. It is turned on in prod, but it is turned off by default for test, dev and staging. Can we rephrase this sentence?

Comment (lchrusciel) [reply docs/cookbook/mailer.rst]:
```
**Remember to remove the** `disable_delivery: true` **parameter from your current enviroment**. This parameter is set to true by default in dev, test and staginig?
```

Comment (CoderMaggie) [reply docs/cookbook/mailer.rst]: That is why I advise to remove it from config_**prod** :)

File: docs/cookbook/mailer.rst (hunk 1 above)
Comment (CoderMaggie) [reply docs/cookbook/mailer.rst]: Fixed.

**Comment by pjedrzejewski** [comment]
Great work Magda, thank you! :)

**Pull request merged**